packages/carequality-sdk/src/client/carequality-fhir.ts

```diff
@@ -203,13 +190,12 @@ export class CarequalityManagementAPIImpl implements CarequalityManagementAPI {
    * @param oid string containing the organization OID
    * @returns an XML string containing an OperationOutcome resource - see Carequality documentation for details - https://carequality.org/healthcare-directory/OperationOutcome-create-success-example2.xml.html
    */
-  async updateOrganization(org: string, oid: string): Promise<string> {
+  async updateOrganization({ org, oid }: UpdateOrganizationParams): Promise<Organization> {
     const query = new URLSearchParams();
     query.append("apikey", this.apiKey);
-    query.append("_format", XML_FORMAT);
```

Comment:
IIUC, we're removing this because default is json? 
Would it be better to set that explicitly? In the past, xml was the default, and then CQ changed it, causing a disruption on our end

Reply:
- apiKey: now its set as a header
- format: agreed